soh/include/z64.h

```diff
@@ -2035,13 +2035,13 @@ typedef struct ArenaNode {
     /* 0x04 */ size_t size;
     /* 0x08 */ struct ArenaNode* next;
     /* 0x0C */ struct ArenaNode* prev;
-    ///* 0x10 */ const char* filename;
-    ///* 0x14 */ s32 line;
-    ///* 0x18 */ OSId threadId;
-    ///* 0x1C */ Arena* arena;
-    ///* 0x20 */ OSTime time;
-    ///* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
-} ArenaNode; // size = 0x10
+    //* 0x10 */ const char* filename;
+    //* 0x14 */ s32 line;
+    //* 0x18 */ OSId threadId;
+    //* 0x1C */ Arena* arena;
+    //* 0x20 */ OSTime time;
+    //* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
```

Comment:
[nit] Lets keep 3 slashes so the original comment block is preserved
```suggestion
    // /* 0x10 */ const char* filename;
    // /* 0x14 */ s32 line;
    // /* 0x18 */ OSId threadId;
    // /* 0x1C */ Arena* arena;
    // /* 0x20 */ OSTime time;
    // /* 0x28 */ u8 unk_28[0x30-0x28]; // probably padding
```

Reply:
Fixed.